src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Missing header comments for class. All classes should have header comments similar to the convention.

Same for the other classes below.